Gopkg.lock

```diff
@@ -378,6 +384,12 @@
   packages = ["."]
   revision = "316fb6d3f031ae8f4d457c6c5186b9e3ded70435"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/google/uuid"
+  packages = ["."]
+  revision = "dec09d789f3dba190787f8b4454c7d3c936fed9e"
+
```

Comment:
we have a somewhat reasonable id generator https://github.com/fnproject/fn/blob/master/api/id/id.go that doesn't require any imports -- this pool manager appears mostly to be for testing purposes and the linked id format will suffice without need to set the machine id.

Reply:
👍 will change